tsconfig.json

```diff
@@ -14,7 +14,10 @@
     "strict": true,
     "target": "ES2017"
   },
+  "exclude": [
+    "**/*.test.ts"
+  ],
   "include": [
-    "./src/*.ts"
+    "./src/**/*.ts"
```

Comment:
why specifying src folder here but exclude has **/*.test.js? Also unrelated: should we add 'node_modules' to exclude?

Reply:
It's because tests might include test-specific libraries and mocks that are outside of the `src` directory.

The "main" tests for each module can certainly be co-located, but there is a lot of test-related stuff that doesn't have any clear home in the `src` directory. A `test` directory makes sense for that stuff.

Reply:
Hm, to Brad's point though, a `test` directory wouldn't be included in the first place. I'll push an update.

Also, `node_modules` is always excluded, IIRC.

Reply:
Exclude reference: https://www.staging-typescript.org/tsconfig#exclude